Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -165,6 +178,32 @@ void DMainWindow::OnOpen()
 		StartGame(filename);
 }
 
+void DMainWindow::OnBrowse()
+{
+	std::string path = ShowFolderDialog().toStdString();
+	if (!path.empty()){
+		std::vector<std::string>::iterator itResult = std::find(
+				SConfig::GetInstance().m_ISOFolder.begin(),
```

Comment:
Good catch.

Fun fact: that part of the routine is ripped straight from CGameListCtrl::BrowseForDirectory() in DolphinWX. Might want to consider fixing it there as well.